starknet_py/net/full_node_client.py

```diff
@@ -101,6 +104,75 @@ async def get_block_traces(
     ) -> BlockTransactionTraces:
         raise NotImplementedError()
 
+    async def get_events(
+        self,
+        address: Hash,
+        keys: List[Hash],
+        from_block_number: Optional[Union[int, Tag]] = None,
+        from_block_hash: Optional[Union[Hash, Tag]] = None,
+        to_block_number: Optional[Union[int, Tag]] = None,
+        to_block_hash: Optional[Union[Hash, Tag]] = None,
+        follow_continuation_token: bool = False,
+        chunk_size: int = 1,
```

Comment:
Let's make these keyword only maybe

cc @war-in @cptartur

Reply:
Do you mean `chunk_size` and `follow_continuation_token`?